spiketools/sim/neurons.py

```diff
@@ -0,0 +1,118 @@
+"""Simulate cells."""
+
+from spiketools.sim.trials import sim_trial_placefield, sim_skew_trial_placefield, sim_trial_multi_placefields, sim_trial_multi_skew_placefields
+
+########################################################################################################################################################
+########################################################################################################################################################
+
+def sim_neuron_placefield(param_gen, vary_height=False, vary_width=False, vary_place_loc=False):
+    """Simulate cell place fields for a given set of parameters.
+    
+    Parameters
+    ----------
+    param_gen : generator
+        Generator of parameters.
+    vary_height : bool
```

Comment:
these params have default values, making them optional to pass in, so should be described as:
`vary_height : bool, optional, default: False`

Check for this here and throughout all docstrings